web/src/app/modules/linking/modules/linking-dialog/components/model-search-bar/model-search-bar.component.ts

```diff
@@ -27,14 +27,18 @@ export class ModelSearchBarComponent implements OnInit {
         this.selectedModel.emit(model);
     }
 
+    get hasModel(): boolean {
+        return this.model !== undefined;
+    }
+
     @Output()
     private selectedModel = new EventEmitter<CEGModel>();
 
     formatter = (m: CEGModel) => {
         if (m !== null) {
             return m.name;
         } else {
-            return this.translate.instant('noResultsFound');
+            return '❌ ' + this.translate.instant('noResultsFound');
```

Comment:
could there occur any problem when using this icon directly instead of e.g. using font awesome?

Reply:
Hmmm, good point. Actually, I think it's not a question of robustness, since all more or less recent browsers understand UTF-8 and have the symbols to display it. Additionally, we are forced to use a hopelessly outdated version of font awesome, and that made me prefer this way. But I'll change it to FA. This makes it look the same in all browsers.

Reply:
OK, it's a little sad, but changing that is really painful. I'll leave it as it is and wait for issues if users see weird symbols.